spec/avromatic/model/builder_nested_models_spec.rb

```diff
@@ -0,0 +1,65 @@
+describe Avromatic::Model::Builder, 'nested_models' do
```

Comment:
Technically I think it is "additional description" here, not a tag. 

This was an attempt to break up the specs a little from their mostly monolithic form. 

The only way this additional description is used is in the test output (subject to formatter) and aligning the the spec file path, as enforced by rubocop-rspec.